packages/react-native-editor/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@wordpress/react-native-editor",
-	"version": "1.50.1",
+	"version": "1.51.1",
```

Comment:
This diff is a bit puzzling to me. The version in [`trunk` is set to `1.51.0`](https://github.com/WordPress/gutenberg/blob/cebcfb04281f105174303a453bc91cfa8ce3a136/packages/react-native-editor/package.json#L3), so I'm not sure why this diff is showing `1.50.1`.  It likely not a big issue, as the diff show the _correct new version_. It must be a side effect of how this beta fix is a branch off [the `rnmobile/1.51.0` tag](https://github.com/WordPress/gutenberg/releases/tag/rnmobile%2F1.51.0). Once the merge conflicts for the PR are resolved (after the release is tagged), this should be resolved. 
<img width="557" alt="image" src="https://user-images.githubusercontent.com/438664/116260151-0e9d5380-a73c-11eb-9cbf-7aedda3cf144.png">